jenkins/pipelines/ci/tidb/tidb_ghpr_check_2.groovy

```diff
@@ -108,7 +78,7 @@ def run_with_pod(Closure body) {
             containers: [
                     containerTemplate(
                             name: 'golang', alwaysPullImage: false,
-                            image: "${pod_go_docker_image}", ttyEnabled: true,
```

Comment:
should change `go get gotest.tools/gotestsum` to `go install gotest.tools/gotestsum@latest`